### DISCUSSION TITLE
Installed failed ... cmake error

### DISCUSSION THREAD

**Discussion by javapep** [fontforge/fontforge#4919] [Q&A] [answered]
I get a problem when I wan to install fontforge via cmake. That is happen " python3 missing" even though it's already installed. Thank u in advance

`CMake` Error at /usr/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:146 (message):
  Could NOT find Python3 (missing: Python3_LIBRARIES Python3_INCLUDE_DIRS
  Development) (found suitable version "3.9.10", minimum required is "3.6")
Call Stack (most recent call first):
  /usr/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:393 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake-3.16/Modules/FindPython/Support.cmake:2214 (find_package_handle_standard_args)
  /usr/share/cmake-3.16/Modules/FindPython3.cmake:300 (include)
  cmake/PackageUtils.cmake:41 (find_package)
  CMakeLists.txt:149 (find_package_auto)

-- Configuring incomplete, errors occurred!
See also "/home/javapep/fontforge/build/CMakeFiles/CMakeOutput.log".
See also "/home/javapep/fontforge/build/CMakeFiles/CMakeError.log".`

**Comment by iterumllc** [comment, marked as answer]
This looks like a probably linux distro. Does it have a separate python-dev package for include files (and such)?

**Reply by javapep** [reply]
maybe, i use Linux mint xfce ulyana 20... Coz  I read in cmake guide that problem can be happen on other distro with pyhton 2.7.x (mint use it).. how to fix it ?... But in the page https://github.com/fontforge/fontforge/wiki/CMake-guide-for-FontForge (cmake guide) that mention it to fix it but idk how to compile the syntax.. Thanks
